index.html

```diff
@@ -41,5 +41,163 @@ <h2>I'am a Software Developer</h2>
        </ul>
 
     </section>
+    <section class="recent-work">
+        <h2>My recent works</h2> 
+
+        <hr>
+        
+        <img src="Images/worksection/hero-image.png" alt="image-hero">
+
+        <h3>Multi-Post Stories</h3>
+
+        <p class="text">A daily selection of privately personalized reads; no accounts or
+             sign-ups required. has been the industry's standard dummy 
+             text ever since the 1500s, when an unknown printer took a
+              standard dummy text.</p>
+
+              <ul>
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <a href="#">See project</a>
+
+            </section>
+
+
+    <section class="work-container">
+        <h6>work articles</h6>
```

Comment:
[OPTIONAL] Kindly remove these extra h6 tags. Since they play no part in the visual layout of the project and you have set their style to `display: none;` - they are acting like commented Code 😀